safe-defaults/Makefile.am

```diff
@@ -40,10 +40,12 @@ safe_defaults_DATA = \
 safe_defaults_chrome_policy_safe_searchdir = \
 	$(datadir)/eos-safe-defaults/chrome/policies/managed
 safe_defaults_chrome_policy_safe_search_DATA = \
+	01_doh.json \
 	chrome-safe-search.json \
 	$(NULL)
 
 EXTRA_DIST = \
+	01_doh.json \
```

Comment:
What's the rationale for this naming rather than, e.g. `dns-over-https.json`?

Reply:
How about "doh2opendns.json"?

Reply:
Have it as `dns-over-https.json` in the new commit.